docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -164,14 +164,28 @@ app.
 
 == Debugging
 
-You can run a Quarkus application in debug mode using:
+In development mode, Quarkus starts by default with debug mode enabled, listening to port `5005` without suspending the JVM.
+
+This behavior can be changed by giving the `debug` system property one of the following values:
+
+* `false` - the JVM will start with debug mode disabled
+* `true` - the JVM will start in debug mode and be suspended until a debugger is attached to port `5005`
+* `client` - the JVM will start in client mode and attempt to connect to `localhost:5005`
+* `{port}` - the JVM will start in debug mode and be suspended until a debugger is attached to `{port}`
+
+So, you can run a Quarkus application in debug mode listening to port `5006` instead of the default port `5005` using:
 
 [source,shell]
 ----
-./mvnw compile quarkus:dev -Ddebug=true
+./mvnw compile quarkus:dev -Ddebug=5006
```

Comment:
This was an interesting example as it waits for you to attach your debugger.

Reply:
The new example also waits for it as explained a few lines above. Do you specifically want the -Ddebug=true example in the doc?

Reply:
I think having `-Ddebug` as the first example with everything default would be nice as it's the one that people will use the most.

Defining another port is described above and I don't think it's the most common case.